javascript/packages/cli/src/cli.ts

```diff
@@ -418,6 +420,8 @@ async function spawn(
   // since this shouldn't be a bottleneck in most of the cases,
   // but also can be set with the `-c` flag.
   const spawnConcurrency = opts.spawnConcurrency || 4;
+  const silent = opts.silent || false;
```

Comment:
```suggestion
  const silent = !!opts?.silent;
```

Reply:
`opts` will always exists here since is created by `program.opts()` above.